### PR TITLE
Add comment field for SMB shares

### PR DESCRIPTION
**Pull request by erinix**
NAS-103677

### CODE REVIEW

File: src/app/helptext/sharing/smb/smb.ts [hunk 1]
```diff
@@ -14,6 +15,11 @@ export const helptext_sharing_smb = {
     placeholder_name: T('Name'),
     tooltip_name: T('Enter a name for the share.'),
 
+    placeholder_comment: T("Comment"),
+    tooltip_comment: T(
+      "Set the share comment."
```
Comment (wblock) [inline src/app/helptext/sharing/smb/smb.ts]:
```suggestion
      "Description of the share or notes on how it is used."
```

File: src/app/helptext/sharing/smb/smb.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { Validators } from "@angular/forms";
 export const helptext_sharing_smb = {
     column_name: T('Name'),
     column_path: T('Path'),
+    column_comment: T('Comment'),
```
Comment (wblock) [inline src/app/helptext/sharing/smb/smb.ts]: Network interfaces have a similar field called Description. We should standardize on one name for this type of field, and Description is clearer.

File: src/app/helptext/sharing/smb/smb.ts [hunk 3]
```diff
@@ -14,6 +15,11 @@ export const helptext_sharing_smb = {
     placeholder_name: T('Name'),
     tooltip_name: T('Enter a name for the share.'),
 
+    placeholder_comment: T("Comment"),
```
Comment (wblock) [inline src/app/helptext/sharing/smb/smb.ts]: As above.

Comment (erinix) [reply src/app/helptext/sharing/smb/smb.ts]: See my last commit, I have renamed Comment description in SMB as well as AFP, NFS and WebDav (will make tickets for the other places this should probably be done in)

**Pull request merged**